multigeiger/userdefines-example.h

```diff
@@ -63,3 +63,20 @@
 // 0x2A38: Heart Rate Sensor Position --> sends TUBE_TYPE
 // 0x2A39: Heart Rate Control Point --> allows to reset "energy expenditure", as required by service definition
 #define SEND2BLE false
+
+// Play an alarm sound when radiation level is too high?
+// Activates when either accumulated dose rate reaches the set threshold (s. below)
+// or when the current dose rate is higher than the accumulated dose rate by the set factor (s. below)
+// ! Requires a valid tube type to be set in order to calculate dose rate
+#define LOCAL_ALARM false
+
+// Accumulated dose rate threshold to play the local alarm
+// The accumulated dose rate is the overall average since the last start of MultiGeiger
+// Default value: 0.500 µSv/h
+#define LOCAL_ALARM_THRESHOLD 0.500  // µSv/h
+
+// Factor of current dose rate vs. accumulated dose rate to play the local alarm
+// Default value: 3
+// Example: accumulated dose rate from days of operation is at 0.1 µSv/h
+// If current dose rate rises to (default) 3 times the accumulated dose rate, i.e. 0.3 µSv/h, sound the alarm
+#define LOCAL_ALARM_FACTOR 3  // current / accumulated dose rate
```

Comment:
can we say "trigger the alarm" instead of "play" / "sound" alarm?

i plan to add a LED to signal misc stuff later, so there might be more alarm options due to that.

Reply:
ok